### PR TITLE
 refactor testsuite

### LINKED ISSUE #75: Optimize/Refactor test suite
* [x] upgrade test code base to current standard #73 
* [x] optimize test classes performance and readability
  * [x] `LocalesTest` #77 
  * [x] `ScopesTest` #86
  * [x] `TestCoreModelExtension` #87
  * [x] `TranslatableTest` #84 
  * [x] `ValidationTest` #88 
* [x] Remove temp `TestsBase` class #81 
* [x] Remove `tests/seeds` folder #89 
* [x] reduce amount of models to the absolute minimum #101
* [x] use `namespace Astrotomic\Translatable\Tests` for tests #100 
* [x] use `static::` instead of `$this->` for assertions #91 
* [x] set `void` return type for all test methods #92
* [x] move to TravisCI instead of CircleCI #83
* [x] use sqlite in local environment (default) use MySQL in CI #83
* [x] Add @Lloople to package credits #82

### PR DESCRIPTION
**Pull request by Gummibeer**
fixes #75

### CODE REVIEW

File: tests/models/VegetableTranslation.php [hunk 1]
```diff
@@ -7,4 +7,10 @@
 class VegetableTranslation extends Eloquent
 {
     public $timestamps = false;
+
+    protected $fillable = [
+        'country_id',
```
Comment (Lloople) [inline tests/models/VegetableTranslation.php]: This should be `vegetable_identity` or something like that if I recall correctly 🤔

**Comment by Gummibeer** [comment]
🎉 🎉 🎉 

```
$ vendor/bin/phpunit 
PHPUnit 8.4.1 by Sebastian Bergmann and contributors.

...............................................................  63 / 137 ( 45%)
............................................................... 126 / 137 ( 91%)
...........                                                     137 / 137 (100%)

Time: 9.39 seconds, Memory: 28.00 MB

OK (137 tests, 289 assertions)
```

File: tests/models/VegetableTranslation.php (hunk 1 above)
Comment (Gummibeer) [reply tests/models/VegetableTranslation.php]: https://github.com/Astrotomic/laravel-translatable/pull/76/commits/5d352596e295a48d38a17323e6b839f3e4221e38

**Comment by Lloople** [comment]
```
$ vendor/bin/phpunit
PHPUnit 8.4.1 by Sebastian Bergmann and contributors.

...............................................................  63 / 139 ( 45%)
............................................................... 126 / 139 ( 90%)
.............                                                   139 / 139 (100%)

Time: 2.02 seconds, Memory: 26.00 MB

OK (139 tests, 291 assertions)
```

This is going really fast! 🤩

**Comment by Gummibeer** [comment]
Documentation updated - so v11.6 will be released today. 🎉 
Thanks @Lloople for this amazing suggestion, work and time you spend in this PR! 💙

**Comment by Gummibeer** [comment]
https://github.com/Astrotomic/laravel-translatable/releases/tag/v11.6.0

**Comment by Lloople** [comment]
It was a pleasure! Really helped me to understand deeper how the package works and the `why`'s behind some decissions / features

**Pull request merged**